lib/src/util/dart_type_utilities.dart

```diff
@@ -18,6 +18,11 @@ class DartTypeUtilities {
       (type is InterfaceType &&
           extendsClass(type.superclass, className, library));
 
+  static Element getRealElement(Element element) =>
+      element is PropertyAccessorElement
+          ? getRealElement(element.variable)
```

Comment:
The variable of a PropertyAccessorElement will never be a PropertyAccessorElement, so the recursive call is not necessary.